src/front/Shared/NavMenu.razor

```diff
@@ -1,29 +1,15 @@
 ﻿<div class="top-row ps-3 navbar navbar-dark">
     <div class="container-fluid">
-        <a class="navbar-brand" href="">yamltreeviewer</a>
-        <button title="Navigation menu" class="navbar-toggler" @onclick="ToggleNavMenu">
-            <span class="navbar-toggler-icon"></span>
-        </button>
+        <a class="navbar-brand" href="/">Yaml Tree Viewer</a>
```

Comment:
Instead of use anchor tag, you can use NavLink component which has some handy additional logic in case in the future we want to add more tabs, and it will be smart about context.